test/usage.test.ts

```diff
@@ -446,3 +446,14 @@ test('Another limitation: passing `this` references to `expectTypeOf` results in
   // Instead of the above, try something like this:
   expectTypeOf(B).instance.toEqualTypeOf<{b: string; foo: () => void}>()
 })
+
+test('Overloads limitation: overloaded functions which include an overload looking like `(...args: unknown[]) => unknown` will excluded `unknown[]` from `.parameters` and exclude `unknown` from `.returns`. This overload just allows any input and returns unknown output anyway though.', () => {
```

Comment:
```suggestion
test('Overloads limitation: Overloaded functions that include an overload resembling `(...args: unknown[]) => unknown` will exclude `unknown[]` from `.parameters` and `unknown` from `.returns`. This overload, however, allows any input and returns an unknown output anyway.', () => {
```

Reply:
I removed this in the end/ replaced it with a similar hand-written part of the readme. It's only a limitation that applies on old typescript versions.